src/api/middleware/__tests__/verify_jwt.test.ts

```diff
@@ -0,0 +1,11 @@
+import server from "../../server";
+import request from "supertest";
+
+import StatusCodes from "../../utils/status_codes";
+
+describe("Testing a sample protected route", () => {
```

Comment:
add tests checking the data is coming back in JSON format and also that the shape of the data is correct eg. do we have keys on the response object that we expect to have.